supporting/concentration_layer.py

```diff
@@ -231,6 +241,77 @@ def process_emissions(self, emis, isrm_obj, verbose, output_dir, output_emis_fla
             
         return tmp_dct['PM25'], tmp_dct['NH3'], tmp_dct['VOC'], tmp_dct['NOX'], tmp_dct['SOX']
     
+    def visualize_emissions(self, pollutant_name=''):
```

Comment:
Can you add more comments to this new function? Each little code chunk should have a header, and if there's anything that isn't immediately intuitive, it should be noted